model/src/cfd.rs

```diff
@@ -1328,7 +1328,7 @@ impl Cfd {
     }
 
     pub fn fail_collaborative_settlement(self, error: anyhow::Error) -> CfdEvent {
-        tracing::error!(order_id=%self.id(), "Collaborative settlement failed: {:#}", error);
+        tracing::warn!(order_id=%self.id(), peer_id=?self.counterparty_peer_id, "Collaborative settlement failed: {:#}", error);
```

Comment:
But this PR doesn't add any logs to `model::cfd`. It's only modifying one instance out of 19 pre-existing log statements that were already there.

I'm not against keeping the `model` crate effect-free, but that's not something that we've actually been doing since we introduced it.

Reply:
yeah, totally. I was just wondering if we should rather remove the logs from model crate and keep them in the other crates. It's probably something that we can discuss and then add a specific ticket. Feel free to ignore it for this PR :)

Reply:
Nit on this line: do we want to have some kind of consistent style with logs and tags? Are we going with no spaces between label and value or more like `order_id = %self.id()`

Reply:
> Nit on this line: do we want to have some kind of consistent style with logs and tags? Are we going with no spaces between label and value or more like order_id = %self.id()

Good catch. I like having spaces too, I'll fix it. But anything that cannot be automatically enforced by `rustfmt` is just too hard to keep under control :(